loglist2/loglist2.go

```diff
@@ -136,28 +150,38 @@ type TreeHead struct {
 	TreeSize int64 `json:"tree_size"`
 }
 
-// String method returns printable name of the state.
-func (ls *LogStates) String() string {
+// Which method returns Log-status for descriptive struct.
```

Comment:
`Which` is a little generic. How about `LogStatus` ?

Reply:
Renamed.